heyjoycelee/breakable-toy
# Message every user automatically when the app is installed to a Slack workspace to explain the tool

Created a place for the [slack button](https://api.slack.com/docs/slack-button) to live, so the bot can be added to a workplace. When the bot is added, it will message all users automatically with an `@channel` to let them know it has been added.

How to test:
- go to /slackbot/index
- add slackbot to channel using the "Add to Slack" button
- confirm you get a notification in Slack

Resolves Issue #7
